limesurvey/limesurvey.py

```diff
@@ -148,11 +148,22 @@ def get_student_access_code(self, anonymous_user_id):
 
         return response.get("result").get("token")
 
-    def add_participant_to_survey(self, user, anonymous_user_id):
+    def add_participant_to_survey(self, user, anonymous_user_id: str):
         """
         Add the student as participant to specified survey.
+
+        args:
+            user: The user to add as participant
+            anonymous_user_id: The anonymous user id of the user
         """
-        firstname, lastname = user.profile.name.split()
+        firstname, lastname = None, None
+
+        if user.profile.name is not None:
+            fullname = user.profile.name.split()
+            if len(fullname) > 1:
+                firstname, lastname = fullname
+            else:
+                firstname = fullname[0]
```

Comment:
```suggestion
        first_name, last_name = "", ""
        if user.profile.name:
           fullname = user.profile.name.split(" ", 1)
           first_name = fullname[0]
        if fullname[1:]:
             last_name = fullname[1] 
```
can we move this and use it as a static method?

Reply:
Done, I created a new method called `get_fullname()`